ledger/evalIndexer.go

```diff
@@ -39,6 +39,8 @@ type indexerLedgerForEval interface {
 	GetAssetCreator(map[basics.AssetIndex]struct{}) (map[basics.AssetIndex]FoundAddress, error)
 	GetAppCreator(map[basics.AppIndex]struct{}) (map[basics.AppIndex]FoundAddress, error)
 	Totals() (ledgercore.AccountTotals, error)
+	// BlockTimeStamp returns timestamp for a block up to TxnMaxLife-1 back from Latest, or 0 if not exist
```

Comment:
Comment should say "+1" since it's talking about going "back".

Reply:
done